bytestring_splitter/__init__.py

```diff
@@ -156,17 +156,24 @@ def __mul__(self, times_to_add):
 
         return new_splitter
 
-    def repeat(self, splittable):
+    def repeat(self, splittable, as_set=False):
```

Comment:
Yeah, perhaps this can evolve into a factor function? Otherwise if it's always a `list` or a `set` - perhaps the name `unique=True`